docs/pages/machine-workload-identity/machine-id/deployment/deployment.mdx

```diff
@@ -76,6 +76,7 @@ integration and continuous deployment platform
 |--------------------------------------------------------------------------------------------------------|---------------------------------------------------------------|------------------------------------------|
 | [Azure DevOps](azure-devops.mdx)                                                                       | TAR archive                                                   | Azure DevOps-signed identity document    |
 | [Bitbucket Pipelines](bitbucket.mdx)                                                                   | TAR archive                                                   | Bitbucket-signed identity document       |
+| [Bound Keypair static key][bound-keypair-static]                                                       | Package manager or TAR archive                                | Bound Keypair static key                 |
```

Comment:
Should we add a remark under the `CI/CD` heading that states basically "Hey Bound Keypair static key is the fallback you should use if your provider isn't explicitly in this list"

Reply:
Good call, I've added a note below the table.